src/components/layout/Finishable/FinishableContainer.js

```diff
@@ -2,26 +2,29 @@ import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import { compose } from 'redux'
 import Finishable from './Finishable'
+import getIsBooked from '../../../utils/getIsBooked'
 import { selectStockById } from '../../../selectors/data/stocksSelectors'
-import { selectBookingById } from '../../../selectors/data/bookingsSelectors'
+import { selectBookingByRouterMatch } from '../../../selectors/data/bookingsSelectors'
 import { selectOfferById } from '../../../selectors/data/offersSelectors'
 
 export const mapStateToProps = (state, ownProps) => {
   const { match } = ownProps
   const { params } = match
   const { bookingId, offerId: offerIdQueryParam } = params
-
+  const booking = selectBookingByRouterMatch(state, match)
+  const isBookedByCurrentUser = getIsBooked(booking)
   let offerId = offerIdQueryParam
+
   if (bookingId) {
-    const booking = selectBookingById(state, bookingId)
     const { stockId } = booking
     const stock = selectStockById(state, stockId)
     const { offerId: offerIdFromStock } = stock
     offerId = offerIdFromStock
   }
+
   const offer = selectOfferById(state, offerId) || {}
-  const isOfferTuto = Object.keys(offer).length == 0
-  const isOfferBookable = !isOfferTuto && offer.isBookable
+  const isOfferTuto = Object.keys(offer).length === 0
+  const isOfferBookable = offer.isBookable || isBookedByCurrentUser
```

Comment:
je ne suis pas fan d'appeler cette variable "isOfferBookable" vu que tu regardes si l'offre est bookable OU est déjà booked. Le nom porte donc à confusion pour moi.
```suggestion
  const isOfferBookableOrBooked = offer.isBookable || isBookedByCurrentUser
```

Reply:
👍